### ISSUE TITLE
Does this project work now with new FB api changes?

### ISSUE THREAD

**Issue by gkakasevski** [jpryda/facebook-multithreaded-mass-scraper#1]
As I know, to collect data from public pages you need a app with permission from FB?

**Comment by nredd** [comment]
Correct, it doesn't seem like you can do any scraping on public pages which you do not own/have admin access to.
